docs/src/documentation/03-components/04-overlay/dialog/03-accessibility.mdx

```diff
@@ -0,0 +1,138 @@
+---
+title: Accessibility
+redirect_from:
+  - /components/dialog/accessibility/
+---
+
+## Accessibility
+
+The Dialog component has been designed with accessibility in mind, providing features that enhance the experience for users of assistive technologies.
+
+### Accessibility props
+
+The following props are available to improve the accessibility of your Dialog component:
+
+| Name        | Type                                                    | Description                                                                                                                     |
+| :---------- | :------------------------------------------------------ | :------------------------------------------------------------------------------------------------------------------------------ |
+| title       | `React.Node`                                            | Provides a visible title that is associated with the dialog as its accessible name.                                             |
+| titleAs     | `"h1" \| "h2" \| "h3" \| "h4" \| "h5" \| "h6" \| "div"` | Defines the semantic HTML element for the title while maintaining its visual appearance.                                        |
+| description | `React.Node`                                            | Provides additional context about the dialog's purpose, which is automatically associated with the dialog via aria-describedby. |
+| onClose     | `() => void \| Promise`                                 | Callback that is triggered when the dialog is closed, supporting keyboard interaction and screen readers.                       |
+| triggerRef  | `React.RefObject<HTMLElement>`                          | References the element that triggered the dialog, allowing focus to return to it when the dialog closes.                        |
+
+### Automatic Accessibility Features
+
+The Dialog component automatically implements the following accessibility features:
+
+- `role="dialog"`: Identifies the element as a dialog to assistive technologies.
+
+- `aria-modal="true"`: Indicates that the dialog is modal, meaning it blocks interaction with page content.
+
+- `aria-labelledby`: Automatically associates the dialog with its title for screen readers using a generated ID.
```

Comment:
Looking at the implementation, I'm not sure if this is true. It's pointing to another child's div wrapper and not the title element.